pkg/integration/tests/bisect/basic.go

```diff
@@ -14,7 +14,7 @@ var Basic = NewIntegrationTest(NewIntegrationTestArgs{
 			NewBranch("mybranch").
 			CreateNCommits(10)
 	},
-	SetupConfig: func(cfg *config.AppConfig) {},
+	SetupConfig: func(cfg *config.AppConfig) { cfg.UserConfig.Git.Log.ShowGraph = "never" },
```

Comment:
Could we have this on its own line? Just so that the diff is cleaner when another line is eventually added (same with the other instances)

Reply:
Sure, changed in 7ae1b3b0686b.